### ISSUE TITLE
pandoc-crossref 0.3.13.0 incompatible with pandoc 2.19.2

### ISSUE THREAD

**Issue by xot** [Homebrew/brew#14243]
### `brew doctor` output

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  python@2

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  cvsps

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libDellMonitorSdkLib.dylib
  /usr/local/lib/libopenh264-1.4.0-osx64.dylib

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  python@2
  ghostscript
  python@3.10
  unbound
```

### Verification

- [X] My "`brew doctor` output" above says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` twice and am still able to reproduce my issue.
- [X] This issue's title and/or description do not reference a single formula e.g. `brew install wget`. If they do, open an issue at https://github.com/Homebrew/homebrew-core/issues/new/choose instead.

### `brew config` output

```shell
HOMEBREW_VERSION: 3.6.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: 001bacee18c512657e1535dff757c8cfdaefc116
Last commit: 8 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: ae2623174177b71512069234040690472682be17
Core tap last commit: 3 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 14.0.0 build 1400
Git: 2.38.2 => /usr/local/bin/git
Curl: 7.84.0 => /usr/bin/curl
macOS: 13.0.1-x86_64
CLT: 14.1.0.0.1.1666437224
Xcode: N/A
```

### What were you trying to do (and why)?

I am trying to process a markdown document containing citations with pandas and pandoc-crossref. I have the latest versions of panda (2.19.2) and pandoc-crossref (0.3.13.0) installed. I run

pandoc --pdf-engine=latexmk --wrap=preserve --filter pandoc-crossref --biblatex -N -o test.pdf test.md

### What happened (include all command output)?

WARNING: pandoc-crossref was compiled with pandoc 2.18 but is being run through 2.19.2. This is not supported. Strange things may (and likely will) happen silently.

This happens even though https://formulae.brew.sh/formula/pandoc-crossref#default says that it depends on panda 2.19.2

### What did you expect to happen?

I did expected pandoc-crossref to be compiled with the latest version of pandoc.

In other words: I expect the version of pandoc-crossref as available through home-brew to always be compatible with the latest version of pandoc as provided by home-brew.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
N.A.
```
